google-maps/README.md

```diff
@@ -22,9 +22,14 @@ The Google Maps SDK supports the use of showing the users current location via `
 
 Read about [Configuring `Info.plist`](https://capacitorjs.com/docs/ios/configuration#configuring-infoplist) in the [iOS Guide](https://capacitorjs.com/docs/ios) for more information on setting iOS permissions in Xcode.
 
-> The main Google Maps SDK now supports running on simulators on Apple Silicon Macs, however, a dependency from Google called [Google-Maps-iOS-Utils](https://github.com/googlemaps/google-maps-ios-utils) does not yet support Apple Silicon. This is requires a new release of the library from Google. If you are developing on an Apple Silicon Mac, building and running on physical devices is still supported and is the recommended approach.
+> The main Google Maps SDK now supports running on simulators on Apple Silicon Macs, but make sure you are getting latest version of [Google-Maps-iOS-Utils](https://github.com/googlemaps/google-maps-ios-utils) by running `pod update Google-Maps-iOS-Utils` from `ios/App/` folder:
 
-> A workaround on Apple Silicon Macs if you want/need to use the simulator is adding the following line to your Podfile in the `target 'App'` section after the `# Add your Pods here` line:
+```
+cd ios/App
+pod update Google-Maps-iOS-Utils
+```
+
+If you added the previous workaround you getting the unreleased version, you can delete it now by removing this line from `ios/App/Podfile`:
```

Comment:
We should move this above running `pod update` since it should be done first or this one will just be re-installed.

Reply:
I've moved it up.
I didn't do it at first because new users will not have that workaround, so can be confusing for them, but if you think it could cause issues better put it first.